examples/bri-3/src/bri/zeroKnowledgeProof/components/blockchain/blockchain.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable, OnModuleInit } from '@nestjs/common';
+import { ZeroKnowledgeProofVerificationInput } from '../../models/zeroKnowledgeProofVerificationInput';
+
+@Injectable()
+export class BlockchainService implements OnModuleInit {
+  async onModuleInit() {
+    // TODO: connect to blockchain
+  }
+
```

Comment:
it needs a connection url as a constant, smart contract address as a constant (later to be read from configuration). Maybe this can be added as a separate github issue and implemented as part of a separate PR?